Sources/ComposableCoreMotion/Live.swift

```diff
@@ -0,0 +1,255 @@
+import Combine
+import ComposableArchitecture
+import CoreMotion
+
+extension MotionManager {
+  public static let live = MotionManager(
+    accelerometerData: { id in
+      requireMotionManager(id: id)?.accelerometerData.map(AccelerometerData.init)
+    },
+    attitudeReferenceFrame: { id in
+      requireMotionManager(id: id)?.attitudeReferenceFrame ?? .init()
+    },
+    availableAttitudeReferenceFrames: {
+      CMMotionManager.availableAttitudeReferenceFrames()
+    },
+    create: { id in
+      .fireAndForget {
+        if managers[id] != nil {
+          assertionFailure(
+            """
+            You are attempting to create a motion manager with the id \(id), but there is already
+            a running manager with that id. This is considered a programmer error since you may
+            be accidentally overwriting an existing manager without knowning.
+
+            To fix you should either destroy the existing manager before creating a new one, or
+            you should not try creating a new one before this one is destroyed.
+            """)
+        }
```

Comment:
I've made the `MotionManager` a bit more strict than I did for `LocationManager`, and if we like this style then I may update the location library too.